tests/functional/object/mcg/test_mcg_replication_target_unreachable.py

```
@@ -394,3 +396,398 @@ def test_noobaa_replication_target_unreachable_underlying_bucket_deleted(
         _wait_for_replication_alert(
             threading_lock, source_bucket.name, timeout=600, sleep=10, cleared=True
         )
+
+    @tier3
+    @polarion_id("OCS-8034")
+    def test_noobaa_replication_target_unreachable_multi_source_multi_target(
```

Ruff (0.15.20)

[warning] Too many statements (64 > 50)

(PLR0915)